content/applications/finance/accounting/receivables/customer_invoices/epc_qr_code.rst

```diff
@@ -1,68 +1,58 @@
-============================
-Add EPC QR Codes to invoices
-============================
+============
+EPC QR codes
+============
 
-European Payments Council Quick Response Code,
-or **EPC QR Code**, are two-dimensional barcodes
-that customers can scan with their **mobile banking
-applications** to initiate a **SEPA Credit Transfer
-(SCT)**, and pay their invoices instantly.
+European Payments Council quick response codes, or **EPC QR codes**, are two-dimensional barcodes that
+customers can scan with their **mobile banking applications** to initiate a **SEPA credit transfer
+(SCT)** and pay their invoices instantly.
 
-In addition to bringing ease of use and speed,
-it greatly reduces typing errors that would potentially
-make for payment issues.
+In addition to bringing ease of use and speed, it greatly reduces typing errors that would
+potentially make for payment issues.
 
 .. note::
-   This feature is only available in several European
-   countries such as Austria, Belgium, Finland, Germany,
-   and The Netherlands.
+   This feature is only available in several European countries such as Austria, Belgium, Finland,
+   Germany, and The Netherlands.
 
 Configuration
 =============
 
-Go to :menuselection:`Accounting --> Configuration --> Settings`
-and activate the **SEPA QR Code** feature.
+Go to :menuselection:`Accounting --> Configuration --> Settings` and activate the :guilabel:`QR
+Codes` feature.
 
-.. image:: epc_qr_code/epc_qr_code01.png
-   :align: center
-
-Configure your Bank Account’s journal
+Configure your bank account’s journal
 -------------------------------------
 
-Make sure that your *Bank Account* is correctly configured
-on Odoo with your IBAN and BIC.
+Make sure that your :guilabel:`Bank Account` is correctly configured on Odoo with your IBAN and BIC.
 
-To do so, go to :menuselection:`Accounting --> Configuration
---> Journals`, open your *bank journal*, then fill out the
-*Bank Account* and *Bank* under the *Bank Account* tab.
+To do so, go to :menuselection:`Accounting --> Configuration --> Journals`, open your bank journal,
+then fill out the :guilabel:`Account Number` and :guilabel:`Bank` under the :guilabel:`Bank Account
+Number` column.
 
-.. image:: epc_qr_code/epc_qr_code02.png
-   :align: center
+.. image:: epc_qr_code/bank-journal.png
+   :alt: Bank account number column in the bank journal
 
-Issue Invoices with EPC QR Codes
+Issue invoices with EPC QR codes
 ================================
 
-EPC QR Codes are added automatically to your invoices, as
-long as you issue them to customers that are located in a
-country where this feature is available.
+EPC QR codes are added automatically to your invoices, as long as you issue them to customers that
+are located in a country where this feature is available.
```

Comment:
EPC QR codes are added automatically to your invoices. Customers located in Belgium xxxxx will be able to scan and pay them.

Reply:
I went with "EPC QR codes are added automatically to your invoices. Customers whose bank supports making payments via EPC QR codes will be able to scan the code and pay the invoice." Let me know if you like that or have any feedback :)